modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/http/http.js

```diff
@@ -220,20 +220,17 @@ exports.Client = function () {
   }
 
   function getUrlParametersFromTupel(tupelParameters) {
-    var queryParameterPairs = [];
-    for (var i = 0; i < tupelParameters.length; i++) {
-      var name = tupelParameters[i].name;
-      var value = tupelParameters[i].value;
+    const queryParameterPairs = [];
+    const tupleClass = new web.TupelList(tupelParameters, true);
```

Comment:
Isn't `tupelParameters` an instance of `TupelList` already?

Reply:
Probably but we covert it through the new getAll() method of TupelList to be sure data is in desired format.

Reply:
What do you mean by probably? This is part of our code, it should be known what type of objects end here. Such conversions are unnecessary and make the code harder to read and understand. Check my other comment about the TupelList constructor.